Header header

float32 RPS
float32 RPM
float32 Turns
float32 Angle
int32 Ticks
